Homework #1 - Autobiography

nadiahwong opened the pull request:
I thoroughly went through the Setup tab to set up Git within RStudio, as well as relying on the Homework Workflow page, with a very helpful video demonstration to make sure I got it right.
I was surprised how much I struggled to set up Git and GitHub, as well as feeding it through RStudio. Embarrassingly, I got stuck at the part of the Setup where the shell asked for my password and I was confused as to why I could not see what I was typing haha! I consulted a friend who helped me with this. In addition, I was stumped with the new vocabulary I had to quickly pick up as I could not attend any of the live sessions of class. I figured out how to add in those textual elements was by searching up Markdown Basics, which I found to be the easy part.
Overall, once I managed to get Git set up within RStudio, it became a bit more straightforward

nadiahwong committed:
I added my name to README

nadiahwong committed:
Homework #1 - Autobiography

I thoroughly went through the Setup tab to set up Git within RStudio, as well as relying on the Homework Workflow page, with a very helpful video demonstration to make sure I got it right.
I was surprised how much I struggled to set up Git and GitHub, as well as feeding it through RStudio. Embarrassingly, I got stuck at the part of the Setup where the shell asked for my password and I was confused as to why I could not see what I was typing haha! I consulted a friend who helped me with this. In addtion, I think I struggled partly because I was stumped with the new vocabularly I had to quickly pick up as I could not attend any of the live sessions of class. I figured out how to add in those textual elements was by searching up Markdown Basics.

VivianQian19 commented on README.md:
```diff
@@ -1 +1,27 @@
+## Who am I?
+
+I was born on 6 July 1997 in sunny Singapore
+This is what I look like: ![*Premature graduation photos because we probably would not be back together again soon :-(*](/Users/nadiah/Desktop/grad photos/DSC03848-2)
```
I'm not sure if you are using R locally or on the server. If you are using the server, I think the picture didn't work because it was not uploaded onto your working space on R.

nadiahwong replied:
Thanks for pointing that out, Vivian! I just edited and the photo should work. Let me know if it does not! I am editing README.md locally on my computer in RStudio, and then committing in it, and pushing the commit into GitHub. Just want to clarify: what do you mean by server?

VivianQian19 commented on README.md:
```diff
@@ -1 +1,27 @@
+## Who am I?
+
+I was born on 6 July 1997 in sunny Singapore
+This is what I look like: ![*Premature graduation photos because we probably would not be back together again soon :-(*](/Users/nadiah/Desktop/grad photos/DSC03848-2)
```
good work using italics!

VivianQian19 commented on README.md:
```diff
@@ -1 +1,27 @@
+This is what I look like: ![*Premature graduation photos because we probably would not be back together again soon :-(*](/Users/nadiah/Desktop/grad photos/DSC03848-2)
+
+
+### Interests
```
nice work using subheaders

VivianQian19 commented on README.md:
```diff
@@ -1 +1,27 @@
+
+
+### Interests
+1. Party Tricks
+    + I can do a headstand and fold my tongue to look like a three leaf clover
+2. Hobbies
+    + I love to play the guitar
+    + Photography
+    + Acting
```
Great work in using lists

VivianQian19 commented on README.md:
```diff
@@ -1 +1,27 @@
+
+
+### Interests
+1. Party Tricks
+    + I can do a headstand and fold my tongue to look like a three leaf clover
+2. Hobbies
+    + I love to play the guitar
+    + Photography
+    + Acting
```
For lists, you can also use bullet points:
* Party Tricks
   + I can ....

bensoltoff replied:
And for enumerated lists, you don't even have to number the elements yourself. Markdown does it automatically

```
1. Party Tricks
    + I can do a headstand and fold my tongue to look like a three leaf clover
1. Hobbies
    + I love to play the guitar
    + Photography
    + Acting
```

1. Party Tricks
    + I can do a headstand and fold my tongue to look like a three leaf clover
1. Hobbies
    + I love to play the guitar
    + Photography
    + Acting

VivianQian19 commented on README.md:
```diff
@@ -1 +1,27 @@
+
+## Who am I?
+
+I was born on 6 July 1997 in sunny Singapore
```
You can add two spaces after "Singapore" to start a new paragraph

nadiahwong committed:
Editing Image problem that ViviannQian19 mentioned

nadiahwong committed:
Troubleshooting: figuring out how to display a photo

nadiahwong committed:
Add files via upload

nadiahwong committed:
Update README.md

nadiahwong reviewed (commented):
What I did was to go to the forked hw01 repo on my end, where I uploaded the photo of myself. Then, I right clicked on the file and copied the link address. I added that URL address in my README.md and I think the photo should display now

nadiahwong committed:
Troubleshooting again with figuring out how to embed image

nadiahwong committed:
Troubleshooting

Merge branch 'master' of https://github.com/nadiahwong/hw01

# Conflicts:
#	README.md

nadiahwong committed:
Troubleshooting one more time

bensoltoff commented on README.md:
```diff
@@ -1 +1,28 @@
+## Who am I?
+
+I was born on 6 July 1997 in sunny Singapore!
+This is what I look like: ![Japan 2019 during Winter Break](<https://github.com/nadiahwong/hw01/blob/master/me.png>)
```
Generally a better approach to embed images on a separate line in the document. Avoids potential formatting issues in the future. Also you don't need to wrap the URL with `<>` inside of the `()`

```
This is what I look like:

![Japan 2019 during Winter Break](https://github.com/nadiahwong/hw01/blob/master/me.png)
```

bensoltoff replied:
And since the image is in the same repo as the readme, you can even shorten it to `![Japan 2019 during Winter Break](me.png)`

bensoltoff reviewed (commented):
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |         x     |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |           |        x      |            |
| **Achievement, creativity** |           |       x       |            |
| **Reproducibility**         |           |       x       |            |

* Avoid using the "upload files to GitHub" feature. Better to make all changes locally on your computer/RStudio Server. Helps avoid your repo getting out of sync with the version on GitHub
* Uses several Markdown elements
* Good practice with the homework workflow
